api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/ObservationTranslatorImpl.java

```diff
@@ -45,7 +47,7 @@
 
 @Component
 @Setter(AccessLevel.PACKAGE)
-public class ObservationTranslatorImpl implements ObservationTranslator {
+public class ObservationTranslatorImpl extends BaseReferenceHandlingTranslator implements ObservationTranslator {
```

Comment:
Thank you for your time @ibacher to look at this.
I am extending this here so as to access createLocationReferenceByUuid(....) method. I hope I have violated no rule.

Reply:
No! That's great!